eng/pipelines/templates/steps/generate-project-list-and-cache-maven-repository.yml

```diff
@@ -33,6 +33,15 @@ steps:
       ServiceDirectory: ${{parameters.ServiceDirectory}}
       ExcludePaths: ${{parameters.ExcludePaths}}
 
+  - pwsh: |
+    $temp = $(ArtifactPackageNames)
```

Comment:
surround this in quotes if not defined it'll try to invoke this as a cmd I _think_.